ext/LegendSpecFitsRecipesBaseExt.jl

```diff
@@ -1390,7 +1390,7 @@ end
         # 2D histogram for LQ Cut
         xlabel := "Energy"
         ylabel := "LQ (A.U.)"
-        nbins := (0:6:3000, -2.0:0.012:4.0)
+        nbins := (0:6:3000, -0.1:0.0006:0.2)
```

Comment:
Can you do the binning dynamically at least for the `LQ`? I encountered already several issues with extreme RAM usage during the dataflow when algorhtihms are binning "non-normal" data into extremely fine bins which allocates a lot of memory?

Reply:
Well, it looks like here it is always the same "fineness" of binning and the only change is dividing the LQ values by 20 (why is that?)